packages/web/src/scss/components/Header/_Header.scss

```diff
@@ -244,6 +244,8 @@ button.Header__link--current {
     }
 }
 
```

Comment:
I prefer adding no new line after block when reenabling stylelinting.

Reply:
You're right, fixed.